via/services/youtube_api/models.py

```diff
@@ -0,0 +1,118 @@
+from dataclasses import dataclass
+from typing import List, Optional
+
+
+@dataclass
+class VideoDetails:
+    id: str
+    title: str
+    short_description: str
+    author: str
+    thumbnails: List[dict]
```

Comment:
There are other things in here which might be of interest to us. For example:

 * `isPrivate` - Might this be an indicator a user has messed up? We might want to catch that
 * `isLiveContent` - Doesn't seem annotatable?